docs/source/pythonapi/examples.rst

```diff
@@ -0,0 +1,25 @@
+----------------------------------------
+:mod:`openmc.examples` -- Example Models
+----------------------------------------
+
+Simple Models
+-------------
+
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+   :template: myfunction.rst
+
+   openmc.examples.slab_mg
+
+Reactor Models
+--------------
+
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+   :template: myfunction.rst
+
+   openmc.examples.pwr_assembly
```

Comment:
I realize these are ordered alphabetically for a reason, but it might be worth considering adding them in order of complexity (esp. if we foresee adding ever more models).